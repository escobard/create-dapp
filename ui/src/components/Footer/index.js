import React from "react";

import "./styles.scss"

const Footer = () => (
  <footer>
    <p>
      Copyright &copy; 2019
    </p>
  </footer>
);

export default Footer;
