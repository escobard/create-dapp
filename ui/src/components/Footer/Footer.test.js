import React from "react";
import { shallow } from "enzyme";
import Footer from "./index";

describe("Footer snapshot renders", () => {
  it("should render correctly mode", () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
