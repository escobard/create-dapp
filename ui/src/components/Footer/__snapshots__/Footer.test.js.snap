// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot renders should render correctly mode 1`] = `
<footer>
  <p>
    Copyright Â© 2019
  </p>
</footer>
`;
